email_alert_service/config.rb

```diff
@@ -16,18 +16,22 @@ def app_root
     end
 
     def rabbitmq
-      all_configs = YAML.load(ERB.new(File.read(app_root+"config/rabbitmq.yml")).result)
+      # rubocop:disable Security/YAMLLoad
+      all_configs = YAML.load(ERB.new(File.read(app_root + "config/rabbitmq.yml")).result)
+      # rubocop:enable Security/YAMLLoad
       environment_config = all_configs.fetch(environment)
 
       @rabbitmq ||= symbolize_keys(environment_config).freeze
     end
 
     def redis_config
-      symbolize_keys(YAML.load(ERB.new(File.read(app_root+"config/redis.yml")).result))
+      # rubocop:disable Security/YAMLLoad
+      symbolize_keys(YAML.load(ERB.new(File.read(app_root + "config/redis.yml")).result))
+      # rubocop:enable Security/YAMLLoad
```

Comment:
can we use safe_load here instead of needing to disable the cop?

Reply:
I tried using `safe_load` but that stops `defaults: &defaults` and `<<: *defaults` in the YAML from working since they are dynamic features.